AdobeStockImage/Model/SaveImagePreview.php

```diff
@@ -113,11 +110,13 @@ public function execute(int $adobeId, string $destinationPath): void
             $path = $this->storage->save($asset->getPreviewUrl(), $destinationPath);
             $asset->setPath($path);
             $this->saveAsset($asset);
+            return $asset->getThumbnailUrl();
```

Comment:
The returned URL should be a Magento URL, i.e. `local.magento.com/media/file.png`. It should be retrieved based on asset path

Reply:
Ok

Reply:
@Nazar65 after changing the frontend approach in https://github.com/magento/adobe-stock-integration/pull/302 the service no longer has to return the url.

It should `return void`

Reply:
Ok i will update.